lib/Dialect/SV/SVOps.cpp

```diff
@@ -44,6 +44,30 @@ LogicalResult sv::verifyInNonProceduralRegion(Operation *op) {
   return failure();
 }
 
+/// Returns the operation registered with the given symbol name with the regions
+/// of 'symbolTableOp'. recurse through nested regions which don't contain the
+/// symboltable trait. Returns nullptr if no valid symbol was found.
+static Operation* lookupSymbolInNested(Operation* symbolTableOp, StringRef symbol) {
```

Comment:
This looks like a good thing to throw into a bind op verifier in a future patch.